lib/function/probability/factorial.js

```diff
@@ -55,31 +55,36 @@ module.exports = function (math) {
 
       n = n.toNumber();   // Should definitely be below Number.MAX_VALUE  
       if (n < smallBigFacs.length) {
-        return BigNumber.convert(smallBigFacs[n]).toSD(math.precision);
+        return BigNumber.convert(smallBigFacs[n]).toSD(config.precision);
       }
 
+      // Be wary of round-off errors
+      var precision = config.precision + 4;
```

Comment:
Actually, this should probably scale with `n` as `n` determined the number of iterations, thus the amount of potential round-off errors.